File: TP2/Python/Punto4/Materia.py
class Materia:
    def __init__(self,nombre,codigo):
        self.__nombre = nombre
        self.__codigo = codigo
    @property
    def nombre(self):
        return self.__nombre
    @property
    def codigo(self):
        return self.__codigo
    @nombre.setter
    def set_nombre(self,nombre):
        self._nombre = nombre
    @codigo.setter
    def set_codigo(self,codigo):
        self._codigo = codigo
    def Imprimir (self):
        print("{}  {}".format(self.__nombre,self.__codigo))

File: TP2/Java/Punto 4/punto4 mora/Materia.java
public class Materia {

private String nombre;
private String codigo;

//constructor
public Materia(String nombre, String codigo){
    this.nombre = nombre;
    this.codigo = codigo;
}

public String getNombre(){
    return this.nombre;
}

public void setNombre(String nombre){
    this.nombre = nombre;
}

public void setCodigo(String codigo){
    this.codigo = codigo;
}


}
